Use --raw-output switch on jq in lieu of xargs

rpdelaney opened the pull request:
Here `xargs` is just stripping the surrounding double quotes, but jq has the --raw-output switch for that.

This has the added benefit that any json string formatting will not be present in the output.

rpdelaney committed:
Use --raw-output switch on jq in lieu of xargs

Here `xargs` is just stripping the surrounding double quotes, but jq has
the --raw-output switch for that.

This has the added benefit that any json string formatting will not be
present in the output.

ralren reviewed (commented):
Looks good. This reminds me I should add more on how to test changes in the README.

I'll approve once I test it.

ralren commented:
Oh, there's some syntax errors.

Assuming credentials are what they should be, you should run the following to see what that's about
```
circle orb validate orb.yml
```

ralren commented:
Testing instructions: https://github.com/trussworks/orb-ecr-image-scan-findings/pull/2

rpdelaney commented:
> Oh, there's some syntax errors.

Odd. I'll check it out.

rpdelaney committed:
Fix unescaped string redirection

Previously this threw an error that '<<' must be escaped in config
versions 2.1+

rpdelaney commented:
I pushed a fix for the validation error. I'll follow up with build test results at a later date.

rpdelaney committed:
Merge branch 'master' into kill_xargs

rpdelaney commented:
I got an `AUTHORIZATION_FAILURE` when attempting to publish the orb for testing :(

ralren commented:
Oh no. I think you need to be a GitHub owner to publish the orb for testing. https://github.com/trussworks/Engineering-Playbook/blob/master/infrasec/circleci/orbs.md#requirements
